packages/jaeger-ui/src/components/TracePage/TraceStatistics/PopupSql.test.js

```diff
@@ -13,27 +13,41 @@
 // limitations under the License.
 
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import PopupSql from './PopupSql';
 
-describe('<PopupSQL', () => {
-  let wrapper;
-  let props;
+describe('<PopupSQL />', () => {
+  const popupContent = 'SELECT * FROM users WHERE id = ?';
+  const closePopupMock = jest.fn();
+
+  const defaultProps = {
+    closePopup: closePopupMock,
+    popupContent,
+  };
 
   beforeEach(() => {
-    props = {
-      closePopup: () => {},
-      popupContent:
-        'select specialtie0_.vet_id as vet_id1_1_0_, specialtie0_.specialty_id as specialt2_1_0_, specialty1_.id as id1_0_1_, specialty1_.name as name2_0_1_ from vet_specialties specialtie0_ inner join specialties specialty1_ on specialtie0_.specialty_id=specialty1_.id where specialtie0_.vet_id=?',
-    };
-    wrapper = shallow(<PopupSql {...props} />);
+    closePopupMock.mockClear();
+    render(<PopupSql {...defaultProps} />);
+  });
+
+  it('renders the header', () => {
+    expect(screen.getByRole('heading', { name: /Tag: "SQL"/i })).toBeInTheDocument();
+  });
+
+  it('renders the SQL content in the textarea', () => {
+    // RTL textarea role implicitly looks for `value` attribute, we need getByDisplayValue
+    // Also checking the value is wrapped in quotes as per the component logic
```

Comment:
is there actually wrapping logic in the component? I don't see it. Wouldn't wrapping be achieved via CSS anyway?

Reply:
This is about it being wrapped in double quotes. I'll update the comment.